src/test/java/net/sf/jabref/logic/util/io/RegExpFileSearchTests.java

```diff
@@ -30,25 +25,23 @@
     @Before
     public void setUp() throws IOException {
 
-        StringReader reader = new StringReader(
-                "@ARTICLE{HipKro03," + "\n" + "  author = {Eric von Hippel and Georg von Krogh}," + "\n"
-                        + "  title = {Open Source Software and the \"Private-Collective\" Innovation Model: Issues for Organization Science},"
-                        + "\n" + "  journal = {Organization Science}," + "\n" + "  year = {2003}," + "\n"
-                        + "  volume = {14}," + "\n" + "  pages = {209--223}," + "\n" + "  number = {2}," + "\n"
-                        + "  address = {Institute for Operations Research and the Management Sciences (INFORMS), Linthicum, Maryland, USA},"
-                        + "\n" + "  doi = {http://dx.doi.org/10.1287/orsc.14.2.209.14992}," + "\n"
-                        + "  issn = {1526-5455}," + "\n" + "  publisher = {INFORMS}" + "\n" + "}");
-
-        BibtexParser parser = new BibtexParser(reader, JabRefPreferences.getInstance().getImportFormatPreferences());
-        ParserResult result = null;
-
-        result = parser.parse();
-
-        database = result.getDatabase();
-        entry = database.getEntryByKey("HipKro03").get();
-
-        Assert.assertNotNull(database);
```

Comment:
Because they are no longer needed (entry and db are always non-null if they are created by hand)